src/types/onyx/BankAccount.ts

```diff
@@ -2,22 +2,38 @@ import type CONST from '@src/CONST';
 import type AccountData from './AccountData';
 import type * as OnyxCommon from './OnyxCommon';
 
+// TODO: This type is a duplicate of the one present in AccountData.ts
+/** Model of additional bank account data */
 type AdditionalData = {
```

Comment:
We have three `AdditionalData` type declarations in the project. Let's remove the other ones and make everyone use this type, also rename it to `BankAccountAdditionalData` and export it.